ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviders.tsx

```diff
@@ -195,6 +215,53 @@ function AuthProviders(): ReactElement {
                     }
                 />
             )}
+            <PageSection>
+                <Alert
+                    isInline
+                    variant="info"
+                    title={
+                        <span>
+                            Consider using short-lived tokens for machine to machine communications
+                            such as CI/CD pipelines, scripts, and other automation.
+                        </span>
+                    }
+                >
+                    <Flex direction={{ default: 'row' }}>
+                        <ExternalLink>
+                            <a
+                                href={getVersionedDocs(
+                                    version,
+                                    'operating/manage-user-access/configure-short-lived-access.html#configure-short-lived-access'
+                                )}
+                                target="_blank"
+                                rel="noopener noreferrer"
+                            >
+                                How to configure short-lived access
+                            </a>
+                        </ExternalLink>
+                        <Link to={`${integrationsPath}/authProviders/machineAccess/create`}>
+                            Create a machine to machine configuration
+                        </Link>
```

Comment:
This needs conditional rendering:

Add above following `hasWriteAccessForPage` assignment.

```ts
const hasWriteAccessForIntegration = hasReadWriteAccess('Integration');
```

Enclose within: `{hasWriteAccessForIntegration && (<Link to={{…}>…</Link>)}`

Although I started to comment about hyphenation, I noticed the integration type: Machine access configuration

So more consistent text: Create a machine access configuration